build.gradle

```diff
@@ -44,7 +44,8 @@ run {
 }
 
 repositories {
-    mavenCentral()
+        mavenCentral()
+        maven { url "http://central.maven.org/maven2" }
```

Comment:
It didn't help either.
Here's what I did:
- I have deleted ~/.gradle directory
- commented the aformentioned line in build.gradle file (central repo url)
- then:
$ cd <project_dir>
$ gradle wrapper
:wrapper

BUILD SUCCESSFUL

$ ./gradlew distZip
Downloading https://services.gradle.org/distributions/gradle-2.2.1-bin.zip
.............................................................
Unzipping /home/<u_name>/.gradle/wrapper/dists/gradle-2.2.1-bin/88n1whbyjvxg3s40jzz5ur27/gradle-2.2.1-bin.zip to /home/<u_name>/.gradle/wrapper/dists/gradle-2.2.1-bin/88n1whbyjvxg3s40jzz5ur27
Set executable permissions for: /home/<u_name>/.gradle/wrapper/dists/gradle-2.2.1-bin/88n1whbyjvxg3s40jzz5ur27/gradle-2.2.1/bin/gradle
:compileJava

FAILURE: Build failed with an exception.

* What went wrong:
Could not resolve all dependencies for configuration ':compile'.
> Could not resolve org.apache.httpcomponents:httpclient:4.3.1.
  Required by:
      pl.touk:sputnik-fork:1.4.1-SNAPSHOT
   > Could not HEAD 'https://repo1.maven.org/maven2/org/apache/httpcomponents/httpclient/4.3.1/httpclient-4.3.1.pom'.
      > hostname in certificate didn't match: <repo1.maven.org> != <edgecastcdn.net> OR <edgecastcdn.net> OR <gp1.wpc.edgecastcdn.net> OR <ne.edgecastcdn.net> OR <minitab.fileburst.com> OR <cdn.montimbrenligne.laposte.fr> OR <images.goldstarbeta.com> OR <images.goldstar.com> OR <content.truste.com [....]

Please let me know if I missed something or should did it in some other way. Thanks.